TrainingExtensions/tensorflow/test/python/test_per_channel_quantization.py

```diff
@@ -256,3 +261,129 @@ def test_qc_quantize_op_cpu_bias(self):
         self.assertTrue(np.allclose(out_data, inp_data))
         sess.close()
 
+    def test_quantsim_created_correctly(self):
+        quantsim_config = {
+            "defaults": {
+                "ops": {
+                    "is_output_quantized": "True",
+                    "is_symmetric": "False"
+                },
+                "params": {
+                    "is_quantized": "True",
+                    "is_symmetric": "True"
+                },
+                "per_channel_quantization": "True",
+            },
+            "params": {},
+            "op_type": {},
+            "supergroups": [],
+            "model_input": {},
+            "model_output": {}
+        }
+
+
+        with open('./quantsim_config.json', 'w') as f:
+            json.dump(quantsim_config, f)
+
+        tf.compat.v1.reset_default_graph()
+        with tf.device('/cpu:0'):
+            model = tf.keras.Sequential()
```

Comment:
please add one testcase for gpu as well. Its ok to add it as part of next pr

Reply:
This is the old PR, which does not have GPU related changes.